site/about/contributing/style-guide/conventions.qmd

```diff
@@ -274,6 +309,53 @@ Within our documentation (`https://docs.validmind.ai/`), you are able to referen
 Refer also to [the glossary](/about/glossary/glossary.qmd) for extended information on product names. 
 :::
 
+## Training materials
+
+The ValidMind Academy is delivered in Revealjs presentation format,[^18] with a slightly different set of conventions:
+
+- It makes use of Tachyons CSS[^19] to provide demonstration overlays.
+
+  Enable Tachyons CSS in the front matter with:
+
+    ```markdown
```

Comment:
```suggestion
    ```yaml
```

Minor nit: the header in Quarto topics is not Markdown, it's YAML metadata. 

(Also "front matter" feels more rooted in the book paradigm but I can see us talking about headers that way. I know I have referred to them that way myself.)

Reply:
Ah, that makes sense, but that's also what Quarto refers to them as in their documentation which is why I went with that since we link out to the Quarto docs heavily.